getting-started/installation-apt-debian.md

```diff
@@ -18,19 +16,32 @@ If you wish to maintain your current version of PostgreSQL outside
 of `apt`, we recommend installing from source.  Otherwise, please be
 sure to remove non-`apt` installations before using this method.
 
+**If you don't already have PostgreSQL installed**, add PostgreSQL's third
+party repository to get the latest PostgreSQL packages:
 ```bash
-# Fetch the correct .deb (e.g., Debian 9 and PostgreSQL 9.6)
-wget https://timescalereleases.blob.core.windows.net/debian/timescaledb-postgresql-9.6_x.y.z~debian9_amd64.deb
+# `lsb_release -c -s` should return the correct codename of your OS
+echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -c -s`-pgdg main" >> /etc/apt/sources.list.d/pgdg.list \
+&& wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add - \
+sudo apt-get update
+```
+
+Add TimescaleDB's third party repository and install TimescaleDB (will download
+any dependencies it needs from PG repo):
+```bash
+# Add our repository
+echo "deb https://packagecloud.io/timescale/timescaledb/debian/ `lsb_release -c -s` main" > /etc/apt/sources.list.d/timescaledb.list
+wget --quiet -O - https://packagecloud.io/timescale/timescaledb/gpgkey | sudo apt-key add -
+sudo apt-get update
 
 # Install via apt-get
-sudo apt-get install ./timescaledb-postgresql-9.6_x.y.z~debian9_amd64.deb
+sudo apt-get install ./timescaledb-postgresql-10_x.y.z~debian9_amd64.deb
```

Comment:
This is no longer correct, you dont deal with .deb files anymore if you setup packagecloud repo. 
You do apt install timescaledb-postgresql-10 or a specific version similar to ubuntu with ppa

Reply:
@svenklemm was correct, I forgot to change these instructions. Should be fixed.